packages/yaml/src/index.js

```diff
@@ -1,11 +1,29 @@
 import YAML from 'js-yaml';
 import toSource from 'tosource';
-import { createFilter, makeLegalIdentifier } from 'rollup-pluginutils';
+import { createFilter, makeLegalIdentifier } from '@rollup/pluginutils';
 
+const defaults = {
+  documentMode: 'single',
+  safe: true,
+  transform: null
+};
 const ext = /\.ya?ml$/;
 
-export default function yamll(options = {}) {
+export default function yamll(opts = {}) {
```

Comment:
Since `opts` is just used with `Object.assign` we can remove the default value.
```suggestion
export default function yamll(opts) {
```

Reply:
This is a stylistic choice. We don't lose anything by defining the default value there.